### DISCUSSION TITLE
[Question] How `initConfig.root` works?

### DISCUSSION THREAD

**Discussion by SoYoung210** [stitchesjs/stitches#751] [Help] [answered]
I have a question about [`initConfig.root`](https://github.com/modulz/stitches/blob/2dde4045f20ee62b2f3b9ee7d8a691edbe8aafbf/packages/core/src/createCss.js#L28) while looking at how to put styles needed in the style tag in stitches.js.

Since there is no `root` as an option of `createCss`, it seems that `globalThis.document` always applied.

**Comment by rob-gordon** [comment, marked as answer]
Just wanted to add that I'm currently doing some hacking where I'm passing a custom `root` (despite typescript not allowing it). Would be cool to get some clarity on whether support for this is planned in the future or if it's out of scope for stitches.
